lib/cocoapods/command/repo/push.rb

```diff
@@ -166,7 +166,7 @@ def validate_podspec_files
         # @return [void]
         #
         def check_repo_status
-          porcelain_status, = Executable.capture_command('git', %w(status --porcelain), :capture => :merge, :chdir => repo_dir)
+          porcelain_status, = Executable.capture_command('git', %w(status --untracked-files=all --porcelain), :capture => :merge, :chdir => repo_dir)
```

Comment:
I don't think we want this change because this would start failing for any spec repo that has untracked files in it, which would not happen today